source/index.html.md

```diff
@@ -177,6 +177,9 @@ error categories, the last two digits define specific errors.
 * 0801: You have hit the request rate-limit
 * 0802: Too many attempts to perform an operation
 
+### Other Errors: 09 (HTTP 420)
+* 0900: Not found users lookup hash
```

Comment:
Why would this just not be a 04xx (validation error) ?

Reply:
I think it is not a validation error, they are trying to retrieve something that is not there, it is more like a business logic error I think.

Reply:
Its the same sort of thing as `Incorrect WID (non-existent or does not belong to user)` though ... they asked for something that isn't there...

Reply:
Fixed